docs/asciidoc/modules/ROOT/partials/usage/apoc.meta.nodes.count.adoc

```diff
@@ -0,0 +1,66 @@
+The examples in this section are based on the following sample graph:
+
+[source,cypher]
+----
+CREATE (n:MyCountLabel {id: 1}), (:MyCountLabel {id: 2}), (m:ThirdLabel {id: 3})
+WITH n,m 
+CREATE (n)-[:MY_COUNT_REL]->(m), (n)-[:ANOTHER_MY_COUNT_REL]->(m), (n)<-[:ANOTHER_MY_COUNT_REL]-(m)
+----
+
+We can return all nodes with a label `MyCountLabel` or a label `ThirdLabel`
+
+[source,cypher]
+----
+RETURN apoc.meta.nodes.count(['MyCountLabel', 'ThirdLabel']) AS count;
+----
+
+.Results
+[opts="header"]
+|===
```

Comment:
I don't think the implementation is what I expected it to be, I expected one result (row) per label